e2etests/tests/evpn_l2.go

```diff
@@ -73,7 +70,14 @@ var _ = Describe("Routes between bgp and the fabric", Ordered, func() {
 		})
 		Expect(err).NotTo(HaveOccurred())
 
-		// TODO create a bridge on each host
+		// Create pre-existing OVS bridges on Kind nodes for testing
+		nodes := []string{infra.KindControlPlane, infra.KindWorker}
```

Comment:
we should be mindful of these as we might want to run the tests in a non kind environment, but it can be deferred later. I general I'd do the node setup work outside of the tests

Reply:
yep, we can defer it (I even wrote it in my comment :P )